test-utils/src/dev/mod.rs

```diff
@@ -17,6 +17,35 @@ use dropshot::ConfigLogging;
 use dropshot::ConfigLoggingIfExists;
 use dropshot::ConfigLoggingLevel;
 use slog::Logger;
+use std::path::Path;
+
+/// Path to the "seed" CockroachDB directory.
+///
+/// Populating CockroachDB unfortunately isn't free - creation of
+/// tables, indices, and users takes several seconds to complete.
+///
+/// By creating a "seed" version of the database, we can cut down
+/// on the time spent performing this operation. Instead, we opt
+/// to copy the database from this seed location.
+pub const SEED_DB_DIR: &str = "/tmp/crdb-base";
+
+// Helper for copying all the files in one directory to another.
+fn copy_dir(
+    src: impl AsRef<Path>,
+    dst: impl AsRef<Path>,
+) -> std::io::Result<()> {
```

Comment:
I'm surprised there's not a standard crate for doing this.  There are a lot of edge cases, which I guess we haven't run into in this case?  (what if there's a named pipe here or a symlink?)

Also, this is all synchronous, even though the caller is async.  I guess that's okay here?